integration-tests/it-producer/src/main/java/org/apache/servicecomb/it/schema/DefaultValueSpringmvcSchema.java

```diff
@@ -64,4 +64,43 @@ public int intHeaderRequire(@RequestHeader(name = "input", required = true, defa
   public int intFormRequire(int input) {
     return input;
   }
+
+  //string
+  @GetMapping("stringQuery")
+  public String stringQuery(@RequestParam(value = "input", defaultValue = "string") String input) {
+    return input;
+  }
+
+  @GetMapping("stringHeader")
+  public String stringHeader(@RequestHeader(value = "input", defaultValue = "string") String input) {
+    return input;
+  }
+
+  @ApiImplicitParams({
+      @ApiImplicitParam(name = "input", dataType = "string", paramType = "form", value = "", defaultValue = "string", required = false)})
+  @PostMapping(path = "stringForm")
+  public String stringForm(String input) {
+    return input;
+  }
+
+  // springmvc rule: required should be false because defaultValue have value
+  @GetMapping(path = "stringQueryRequire")
+  public String stringQueryRequire(
+      @RequestParam(name = "input", required = true, defaultValue = "string") String input) {
+    return input;
+  }
+
+  // springmvc rule: required should be false because defaultValue have value
+  @GetMapping(path = "stringHeaderRequire")
+  public String stringHeaderRequire(
+      @RequestHeader(name = "input", required = true, defaultValue = "string") String input) {
+    return input;
+  }
+
+  @ApiImplicitParams({
+      @ApiImplicitParam(name = "input", dataType = "string", paramType = "form", value = "a required form param", required = true, defaultValue = "string")})
+  @PostMapping(path = "stringFormRequire")
+  public String stringFormRequire(String input) {
```

Comment:
This test case is confusion, because it has two parameters named "input", one is body(json string) and the other is form. How would this work?